docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc

```diff
@@ -106,5 +106,14 @@ Analysis takes minutes to hours to run depending on the size of the application
 ====
 {ProductShortName} relies on Kubernetes scheduling capabilities to determine how many analyzer instances are created based on cluster capacity. If several applications are selected for analysis, by default, only one analyzer can be provisioned at a time.  With more cluster capacity, more analysis processes can be executed in parallel.
 ====
-
 . When analysis is complete, you can click the *Report* link to see the results of the analysis.
+
+[id="viewing-analysis-details_{context}"]
+== Viewing analysis details
+
+You can view the details of an analysis by clicking the options menu ({kebab}) and selecting *Analysis details*. The details are displayed in the *Analysis details for customers* window. You can choose either YAML or JSON format.
```

Comment:
Hi @sbeskin-redhat I feel we are conveying the same thing twice(Repetitive), can't we somehow re-frame both this line in single sentence. `You can view the details of an analysis by clicking the options menu ({kebab}) and selecting *Analysis details*. The details are displayed in the *Analysis details for customers* window. `

Format can be plural

Reply:
@kpunwatk 
I am not sure I understand what you mean. What is repetitive here?

Reply:
@sbeskin-redhat What i meant was this line `The details are displayed in the *Analysis details for customers* window` not sure about this term `Analysis details for customers` if it's something different then it's okay to mention in different sentences. If this means the same window which opens after clicking the Analysis details then i think we could add in same line as it feels `view the details` and `details are displayed` means the same start for the line.

Reply:
@kpunwatk 
In the first sentence, we tell the user which option they should use to view the details - the 'Analysis details' option from the menu.
In the second sentence, we tell the user what happens when they click that option - the 'Analysis details for customers' window opens, and it displays the details.

Reply:
Ah, Okay I see. Thanks for the clarification @sbeskin-redhat 
 I will approve the PR then. Thanks!